sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -183,6 +183,17 @@ async def _client_ready_async(self):  # pylint: disable=no-self-use
         """
         return True
 
+    def __getstate__(self):
+        state = self.__dict__.copy()
+        # remove inherited threading.Lock()
+        state['_mgmt_link_lock'] = None
+        state['_keep_alive_thread'] = None
+        return state
+
+    def __setstate__(self, state):
+        state['_keep_alive_thread'] = asyncio.ensure_future(self._keep_alive_async())
```

Comment:
same with the `_mgmt_link_lock` ?

Reply:
_mgmt_link_lock isn't used on async at all (async uses _mgmt_link_lock_async), so it's not needed here technically